src/status_im2/common/standard_auth/style.cljs

```diff
@@ -0,0 +1,46 @@
+(ns status-im2.common.standard-auth.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(def container
+  {:flex 1})
+
+(def inner-container
+  {:flex 1 :padding-horizontal 20})
+
+(def context-tag
+  {:flex 1 :margin-top 12 :margin-bottom 56 :width 115})
```

Comment:
it's unlikely we want to set the width of the context tag, curious of why you did this?

Reply:
@J-Son89 I actually did it because it's width was full width on the screen. But now I got the issue and fixed it without using `width`.